### ISSUE TITLE
[Bug] Incorrect zoom value shown in the overlay

### ISSUE THREAD

**Issue by syed-qure** [OHIF/Viewers#4077]
### Describe the Bug

The zoom value that is calculated and shown in the ViewportsOverlay component does not match with the viewport.getZoom() value.

### Steps to Reproduce

1. Open a study and not the zoom value in the bottom left corner.
2. Go to console, get the enabledElement and get the zoom value from viewport object
```javasricpt
enabledElement.viewport.getZoom()
```

<img width="1440" alt="Screenshot 2024-05-02 at 10 04 26 AM" src="https://github.com/OHIF/Viewers/assets/109221583/9d9384cf-64e7-4b1c-97af-205a6a0090d6">

### The current behavior

I think there is something wrong in the calculation of scale inside ViewportOverlay's `updateScale` function

### The expected behavior

The zoom value in the ViewportOverlay component should be equal to `viewport.getZoom()`

### OS

macOS 13.2.1 (22D68)

### Node version

18.19.1

### Browser

Arc v1.40.1 (49250)

**Comment by syed-qure** [comment]
Should I just use `viewport.getZoom()` and use its value in setScale inside ViewportOverlay and call it a day?

```typescript
( evt: Types.EventTypes.CameraModifiedEvent ) => {
  const enabledElement = cornerstone.getEnabledElement( evt.element );
  const { viewport } = enabledElement;

  setScale( viewport.getZoom() );
}
```

**Comment by sedghi** [comment]
We released [OHIF 3.9](https://github.com/OHIF/Viewers/tree/release/3.9) which address this issue
